CriticalMass/Result+Additions.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Created for CriticalMaps in 2020
+
+import Foundation
+
+extension Result {
+    func get() throws -> Success {
+        switch self {
+        case let .success(value):
+            return value
+        case let .failure(error):
+            throw error
+        }
+    }
+
+    func isError() -> (Bool, Error?) {
```

Comment:
I'm not sure about this tuple. The call side is a bit unexpected with `result.isError().0` and `result.isError().1`. Maybe we could split them in `var error: Error?` and `hasFailed: Bool`. What do you think about that?

Reply:
think that improves it. I was just a litte lazy I guess. I'll update the signature then to have named parameters or do you think this deserves its own Type?

Reply:
I don't think named parameters will solve the call side problematic `result.isError().isError` and `result.isError().error`... I would split the function into two functions or two separate variables to make it more clear what it's about.

Reply:
I splitted the two up